packages/mdc-menu/adapter.ts

```diff
@@ -57,8 +57,10 @@ export interface MDCMenuAdapter {
 
   /**
    * Closes the menu-surface.
+   * @param skipRestoreFocus Whether to skip restoring focus to the previously
+   *    focused element after the surface has been closed.
    */
-  closeSurface(): void;
+  closeSurface(skipRestoreFocus?: boolean): void;
```

Comment:
to avoid double negation can we name this to `restoreFocus` instead?

Reply:
Since this is an optional param (with default false), and restoring focus is the default behavior in most cases, I prefer leaving this as is so we only need to specify `skipRestoreFocus = true` in the small # of use cases it's actually needed (TAB key for now). Wdyt?

Reply:
Makes sense.. Thanks!